pkg/api/entries.go

```diff
@@ -156,6 +155,21 @@ func CreateLogEntryHandler(params entries.CreateLogEntryParams) middleware.Respo
 		}()
 	}
 
+	payload, err := logEntryAnon.MarshalBinary()
```

Comment:
This is just marshaling to JSON under the covers... we should probably marshal it in a JCS-compatible way before signing so that it is repeatable for clients to verify the hash.  

Also since we support returning data in YAML format I think we need to be explicit in documenting how we format the data before signing since there would be a potential for mismatch there too.

Reply:
Cool, I found this json canonicalization library & added it in: https://github.com/cyberphone/json-canonicalization

Where would be the best spot for the documentation? I could make a `docs` folder in the repo if there's isn't an existing spot for this kind of thing.

Reply:
Nothing really right now outside of the openapi.yaml. I'd try to be as verbose there as you can and then see if we need more space.

Reply:
cool, I added in instructions for how to verify as comments